src/kv/value.rs

```diff
@@ -376,6 +381,9 @@ impl<'v> fmt::Display for Value<'v> {
     }
 }
 
+// REVIEW: I'm wondering if people will accidentally use these implementations
```

Comment:
Good point :+1: I'd be up for removing them.

Reply:
Done in https://github.com/rust-lang/log/pull/585.